quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,141 @@
+
+## About the "Overview" section
+
+Use this section to provide the following:
+
+- A short description of your application and its purpose.
+- A description of what the user can accomplish in this quickstart.
+- The intended audience for this document. 
+- The basic knowledge that you expect the user to have before using this quickstart.
+
+Defining the audience right in the beginning helps the users identify if the quickstart is relevant for their usage.
+
+## About the "Before you begin" section
+
+This section is optional.
+
+Use this section to mention any requirements/configuration prerequisites that the user must complete before they start the quickstart. 
+For easy understanding, we recommend that you classify the prerequisites into different categories such as software prerequisites, hardware prerequisites, and so on. 
+
+
+
+## About the "Installation" section
+
+This section is optional.
+
+:memo: Not all quickstart guides requires installation section. Include this section if:
+      - Installation and/or configuration is  done at the same time, and by the same person running the quickstart.
+      - Installation of a specific software(s) are a prerequisite to running the quickstart. 
+ 
+The purpose of this section is to provide instructions to your users on how to install/configure a particular software/tool before running the quickstart. 
+
+:memo: This section may not be relevant for Cloud/API-based applications where authorization/authentication information is more applicable.
+
+Use this section to provide:
+- Basic instructions/commands to install your application.
+ - Always validate the commands and check for technical accuracy with your engineering team.
+ - Provide instructions to verify that the installation is successful.  
+- Link to the detailed installation guide if applicable.
+- Links to the upstream docs for common software installation instructions.
+
+## About the "Steps" section
+
+The steps section is where you describe what the user needs to do. How you write your steps will vary depending on your organization's style guide.
+This template breaks down the quickstart into **parts**. One part of the quickstart may focus on completing several related steps. You're welcome to follow this structure or use the step headings on their own.
+
+If your quickstart involves many complex tasks then break it down into different logical parts with each part consisting of one or more related steps. For example, structure a quickstart guide to onboard an app as follows:
+Part 1 Configure your local dev environment
+Step 1.1 
+Step 1.2
+Part 2 Build your app
+Step 2.1
+Step 2.2
+Part 3 Deploy your app 
+Step 3.1
+Step 3.2
+Part 4 Test your app 
+Step 4.1
+Step 4.2
+
+However, if your quickstart does not involve many independent tasks, we recommend that you add only the logical steps. For example, structure  a quickstart guide to create a new project in Visual Studio as follows:
```

Comment:
**Nitpick (phrasing, non-blocking):** I again try to avoid using "we" in technical documents. One possible rephrase: "However, if your quickstart does not involve many independent tasks, only add the logical steps."

FYI, there is a double space after "structure."